model/application_authentication.go

```diff
@@ -16,7 +16,6 @@ type ApplicationAuthentication struct {
 	ApplicationID    int64 `json:"application_id"`
 	Application      Application
 	AuthenticationID int64 `json:"authentication_id"`
-	Authentication   Authentication
```

Comment:
This probably shouldn't be getting removed? Not sure why it would be.

Reply:
yes it should stay here, good eye - probably overlooked during rebasing